ferrischat_webserver/src/members/create_member.rs

```diff
@@ -1,6 +1,36 @@
-use actix_web::{HttpResponse, Responder};
```

Comment:
i don't like the idea of being able to join a guild via an endpoint. that is what the invite system is for.

Reply:
+1

I never got around to actually deleting the endpoint, so if you want you can do it now.